### ISSUE TITLE
Doesn't seem to filter links opened in a new tab

### ISSUE THREAD

**Issue by k3a** [KevinRoebert/ClearUrls#49]
### Prerequisites
* [ ] Can you reproduce the problem in a **fresh** firefox profile with **only** ClearURLs?

So you want me to uninstall those 10 other extensions? Sorry won't do.

* [x] Are you running the latest version of ClearURLs?

Yes

* [x] Are you have changed settings under `about:config`?

Yes but I don't remember every single change unfortunately. Btw wording for this question is wrong, isn't it "Are you have"?
### Description

When I visit a link in the email, it calls firefox binary with the link as an argument. That opens a new tab on existing Firefox instance and CleanURLs won't filter utm_source, utm_campaign, etc. The same happens when I create a new tab and paste the full URL there - the original URL is used. Only after I refresh the tab, URL gets cleaned.

In contrast the 'pure-url' extension https://addons.mozilla.org/cs/firefox/addon/pure-url/ is able to filter URLs for new tabs so it looks like a ClearUrls-only problem.

### Steps to Reproduce

1. Click on some stupid marketing link in an email client
2. Link opens as it is in Firefox without removing marketing tracking

**Expected behavior:** 
CleanUrls cleans URLs on new tabs

**Actual behavior:**
URL is not cleaned until I hit refresh.

### System
* **Version**: 1.2.1.4, latest as I have just installed it 
* **Platform**: Linux xxx 4.16.10-1-k3a #2 SMP Tue May 22 00:00:23 CEST 2018 x86_64 GNU/Linux

**Comment by KevinRoebert** [comment]
You do not have to reinstall Firefox or disable all addons. You can just create a new profile that only has ClearURLs installed and where no settings are changed. Instructions can be found here: https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles

Without an example link and without knowing if another addon or your settings may trigger this behavior, it is impossible to fix anything. Please submit the required information so that we can fix the possible bug.

**Comment by k3a** [comment]
After trying the extension in a clear profile it really does work fine (even for new tabs opened externally). So it must be a problem in my configuration or other extensions, sigh. But for some reason the `pure-url` worked even in my environment. 

CLosing for now, I will try to find the real reason later when I have more time and will let you know if I find the cause. Thanks for your prompt response and thanks for making this extension in the first place!

Btw switching to a blank profile is really easy, you should consider adding a shortened link with instructions (or just mentioning "start `firefox` with `-P`") to the issue question. ;)

**Comment by k3a** [comment]
I was unable to determine the exact cause, sorry. :( I simply moved parts of the old profile into a new one (not copying cookies and prefs.js) and it looks like before except that everything works now. :) Sorry for bothering you it really looked like a problem in your extension at first.
